packages/python-dynaconf/python-dynaconf.spec

```diff
@@ -61,12 +58,15 @@ set -ex
 %files -n %{?scl_prefix}python%{python3_pkgversion}-%{pypi_name}
 %license LICENSE
 %doc README.md
-%{_bindir}/dynaconf
+%exclude %{_bindir}/dynaconf
```

Comment:
I think we want `bin/dynaconf` to match what Pulp is using, and its "our" package